server/task_executor_schema_update_ghost_cutover.go

```diff
@@ -16,10 +21,57 @@ func NewSchemaUpdateGhostCutoverTaskExecutor(logger *zap.Logger) TaskExecutor {
 
 // SchemaUpdateGhostCutoverTaskExecutor is the schema update (gh-ost) cutover task executor.
 type SchemaUpdateGhostCutoverTaskExecutor struct {
-	l *zap.Logger //nolint
+	l *zap.Logger
 }
 
 // RunOnce will run SchemaUpdateGhostCutover task once.
 func (exec *SchemaUpdateGhostCutoverTaskExecutor) RunOnce(ctx context.Context, server *Server, task *api.Task) (terminated bool, result *api.TaskRunResultPayload, err error) {
-	return true, &api.TaskRunResultPayload{Detail: "not implemented yet"}, nil
+	defer func() {
+		if r := recover(); r != nil {
+			panicErr, ok := r.(error)
+			if !ok {
+				panicErr = fmt.Errorf("%v", r)
+			}
+			exec.l.Error("SchemaUpdateGhostCutoverTaskExecutor PANIC RECOVER", zap.Error(panicErr), zap.Stack("stack"))
+			terminated = true
+			err = fmt.Errorf("encounter internal error when executing schema update cutover task")
+		}
+	}()
```

Comment:
```suggestion
```
This is no longer needed, the recovery is done by
https://github.com/bytebase/bytebase/blob/b6e043d87d51e079adae8e544f8d8f71485d28ae/server/task_executor.go#L31